# A-discussion-on-differential-expression-and-co-expression-network-analysis using data science and machine learning.
Differential expression analysis of gene expression data and finding the co-expression network and  biological pathway of gene regulation for a specific disease using statistics and data science.
# Differential Expression Analysis.
This technique involves comparing gene expression patterns between different experimental groups to identify genes that exhibit significant differences. It begins with data normalization and employs statistical methods like Wald-tests, negative binomial GLM, or regression models to identify differentially expressed genes (DEGs).
# Gene Co-expression Network Analysis (WGCNA).
Gene co-expression networks are constructed to understand gene interactions and regulatory relationships. WGCNA helps identify hub genes and co-regulated gene modules, providing insights into disease pathogenesis and potential biomarkers.
# Gene Clustering and Hub Gene Identification.
Gene clustering techniques, such as hierarchical and non-hierarchical clustering, are used to identify patterns and relationships in gene expression data. Hub genes, which play crucial roles in regulatory networks, can be identified through centrality analyses.
# Pathway Enrichment Analysis.
This technique identifies biological pathways significantly enriched in a set of genes. It provides insights into the biological mechanisms associated with these genes. GO terms and KEGG pathways are commonly used for this analysis.
# Why this study is important?
The integration of differential expression analysis, pathway enrichment analysis, gene co-expression network analysis, and gene clustering techniques, these analytical approaches facilitate the exploration of gene interactions, regulatory networks, and biological pathways associated with celiac disease, contributing to a deeper understanding of disease mechanisms and potential therapeutic targets.
# Objective of this study.
The primary objective of this study is to leverage RNA-sequencing data to conduct a comprehensive analysis of differential gene expression in individuals affected by celiac disease. This investigation aims to identify and characterize the enriched biological pathways and gene ontology terms associated with celiac disease pathogenesis. Additionally, we intend to construct a weighted gene co-expression network, enabling the identification of hub genes that play pivotal roles in the regulatory mechanisms underlying celiac disease. Ultimately, this research seeks to provide valuable insights into the molecular underpinnings of celiac disease, with the potential to inform future therapeutic interventions and advance our understanding of this complex condition.
# Data Collection 
# Data Preprocessing
# Exploratory data analysis (EDA)
## Principal Component Analysis
## Hierarchical Clustering
# Differential Expression analysis
## Size Factor Estimation
## Estimate Dispersion
## Identification of DEG’s
# Construction of Co-expression Network
## Identification of Clusters
## Identification of Hub and Influential Genes of DEG’s
## Identification of Hub and Influential Genes of Top 20 DEG’s
# Functional and Pathway Enrichment Analysis
## GO Terms Analysis of Common DEGs
## KEGG Pathway Enrichment Analysis of DEGs
